.github/CODEOWNERS

```diff
@@ -0,0 +1,31 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in the repo.
+* @pierre-sassoulas @DanielNoord @cdce8p @jacobtylerwalls @hippo91
```

Comment:
I would suggest to remove the default. It isn't really helpful and will just lead to notification spam.

Codeowners are most useful if they are different from the ones already looking at all PRs anyway.

Reply:
```suggestion
# These owners will be the default owners for everything in the repo.
# Right now there is not default owner to avoid spam
# * @pierre-sassoulas @DanielNoord @cdce8p @jacobtylerwalls @hippo91
```